src/supervisor.tex

```diff
@@ -1815,8 +1817,14 @@ \subsection{Addressing and Memory Protection}
 The N bit indicates that the page represents a
 naturally-aligned power-of-two range of contiguous translations, as defined in
 the Svnapot extension in Chapter~\ref{svnapot}.
+If the Svnapot extension is not implemented, the N bit must be zeroed by software,
+or else a page-fault exception is raised.
 
-Bits 62--54 are reserved
+Bits 62--61 indicate the page-based memory type, as defined in the Svpbmt extension in Chapter~\ref{svpbmt}.
+If the Svpbmt extension is not implemented, the PBMT bits must be zeroed by software,
+or else a page-fault exception is raised.
```

Comment:
Raising a fault breaks compatibility, software will now have to choose a different PTE format based on hw revision, instead of just implementing the latest Priv. Spec with guaranteed backwards compatibility. I raised the same issue here -> https://github.com/riscv/riscv-isa-manual/pull/611#discussion_r650576954

Reply:
Let's resolve this on #611 rather than splitting the discussion in two places